docker/LICENSE

```diff
@@ -0,0 +1,114 @@
+                    Confluent Community License Agreement
```

Comment:
@dnielsen We're creating a Docker image from Confluent's binaries but retaining this license. Please take a look if it's okay.